interface.py

```diff
@@ -68,7 +69,7 @@ def index():
     if request.method == 'POST':
         print(request.form)
         if 'add_file' in request.form and ".." not in request.form['add_file']:
-            item = ('file', request.form['add_file'])
+            item = ('file', request.form['add_file'], datetime.now())
```

Comment:
instead of `datetime.now()` i would use `datetime.now().timestamp()` so that the output to the user is cleaner